src/hooks/cart.js

```diff
@@ -13,8 +13,11 @@ export default class extends BaseHooks {
     }
 
     itemAdd() {
-        this.$body.on('submit', '[data-cart-item-add]', (event) => {
-            this.emit('cart-item-add', event, event.target);
-        });
+        const element = document.querySelector('data-cart-item-add');
+        if (element) {
+            element.addEventListener('submit', (event) => {
```

Comment:
In general, there may be many elements that match, so you'll need to loop over them to add the event listener. Same thing for other files below.

Reply:
Everything has been changed to querySelectorAll and for loops to bind event listeners.